src/dev/run_check_core_api_changes.ts

```diff
@@ -33,23 +33,35 @@ import fs from 'fs';
 import path from 'path';
 import getopts from 'getopts';
 
+/*
```

Comment:
Should we go ahead and rename this file and related CLI commands? Maybe `run_check_published_api_changes`? And for the package.json script: `docs:acceptApiChanges`?

Reply:
How to update CI with the new command name?

Reply:
- https://github.com/elastic/kibana/blob/c477b61e97c4f90ed5013d9d62763ec73e89401a/tasks/config/run.js#L121
- https://github.com/pgayvallet/kibana/blob/c477b61e97c4f90ed5013d9d62763ec73e89401a/tasks/jenkins.js#L27
- https://github.com/pgayvallet/kibana/blob/c477b61e97c4f90ed5013d9d62763ec73e89401a/tasks/test.js#L75